bcda/models/models.go

```diff
@@ -384,7 +403,8 @@ func (cclfBeneficiary *CCLFBeneficiary) GetBlueButtonID(bb client.APIClient) (bl
 	}
 
 	// didn't find a local value, need to ask BlueButton
-	jsonData, err := bb.GetBlueButtonIdentifier(client.HashHICN(cclfBeneficiary.HICN))
+	hashed_hicn := client.HashHICN(cclfBeneficiary.HICN)
```

Comment:
Is snake case intentional or should this be `hashedHICN`?

Reply:
no, I have just been doing some JS on the side.  I will fix it.